coderedcms/templates/coderedcms/pages/form_page.html

```diff
@@ -10,6 +10,13 @@
 <div class="container">
     <form class='{{ page.form_css_class }}' id='{{ page.form_id }}' action="{% pageurl self %}" method="POST" {% if form|is_file_form %}enctype="multipart/form-data"{% endif %}>
         {% csrf_token %}
+
+        {% block captcha %}
+            {% if page.spam_protection %}
+                {% include 'coderedcms/includes/form_honeypot.html' %}
+            {% endif %}
+        {% endblock %}
```

Comment:
Since we are letting people override this, maybe we should put it at the bottom of the form, where a captcha normally would go. Less of a chance the template would have to be forked then just to change the layout if using something like the google captcha instead.